pkg/svc/zserver.go

```diff
@@ -18,9 +18,9 @@ func NewBasicServer(dsn string) (pb.ContactsServer, error) {
 	// In case you didn't have one already.
 	// This is done to make current example up and running
 	// without having contacts DB configured properly.
-	if err := db.AutoMigrate(&pb.ContactORM{}).Error; err != nil {
-		return nil, err
-	}
+	//if err := db.AutoMigrate(&pb.Contact{}).Error; err != nil {
```

Comment:
I vote to just remove this code. We don't want to set a precedent of people automigrating their code. We should instead find a different way for them to set up their database

Reply:
Sounds good. Do we have any process for verifying that the database version and application version are compatible? In lieu of Automigrate, a SchemaCheck might be called for?

Reply:
we don't have any process... we should discuss this as a team at some point :)

Reply:
How will application work without this? We have a DB controller for kubernetes scoped which should be responsible for schema checks and migrations, but that is a big work to do and will not be available in the nearest future.
If remove, please document how to migrate database schema manually, maybe create a `make` target for this.

Reply:
you're still right, Daniil. We still don't have any way to **initialize** a database. So the process still has some manual steps. My concern with using this `automigrate` as a stopgap is that we'll have multiple projects viewing this as the proper way to do this.

So I think our options are:
1. just have it be painful to initialize a db
2. find some other way to do this easily until a proper solution is developed (the DB controller)